### PR TITLE
Add Cloudflare WARP.app Beta Latest (cask cloudflare-warp)

### PR DESCRIPTION
**Pull request by sboh1214**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by sboh1214** [comment]
My bad, didn't realize 1.1.1.1 is a bad token. So I change it to cloudflare-warp.

File: Casks/cloudflare-warp.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask "cloudflare-warp" do
+  version :latest
+  sha256 :no_check
+
+  url "https://1.1.1.1/Cloudflare%20WARP.zip"
```
Comment (miccal) [inline Casks/cloudflare-warp.rb:5]: Please change this to:
```
url "https://one.one.one.one/Cloudflare%20WARP.zip"
```

File: Casks/cloudflare-warp.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  name "Cloudflare WARP"
+  desc "The free app that makes your Internet safer"
+  homepage "https://1.1.1.1/"
```
Comment (miccal) [inline Casks/cloudflare-warp.rb:8]: Please change this to:
```
homepage "https://one.one.one.one/"
```

File: Casks/cloudflare-warp.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  pkg "Cloudflare WARP.pkg"
+
+  uninstall script: {
+    executable: "/Applications/Cloudflare\ WARP.app/Contents/Resources/uninstall.sh",
```
Comment (miccal) [inline Casks/cloudflare-warp.rb:13]: You do not need to escape the spaces here -- please remove the `\`'s.

File: Casks/cloudflare-warp.rb (hunk 2 above)
Comment (reitermarkus) [reply Casks/cloudflare-warp.rb:8]: That's actually the canonical “URL” in this case.

Comment (miccal) [reply Casks/cloudflare-warp.rb:8]: Strange -- earlier today when I checked, `https://1.1.1.1/` was not reachable via `curl` but `https://one.one.one.one/` was fine, hence my initial suggestion for the change. 

@sboh1214 please disregard my suggested changes for the `url` and the `homepage`.

**Comment by ran-dall** [comment]
FMU This is a Beta release, and not entirely stable.

File: Casks/cloudflare-warp.rb (hunk 2 above)
Comment (sboh1214) [reply Casks/cloudflare-warp.rb:8]: @miccal ok I also confirmed that I can reach 1.1.1.1 via curl. Thanks for your reviewing

File: Casks/cloudflare-warp.rb (hunk 3 above)
Comment (sboh1214) [reply Casks/cloudflare-warp.rb]: @miccal Oh! My mistake because of copy&paste
I resolved it in f14be0c. Thanks

**Comment by soneji** [comment]
Could we call this package `cloudflare-warp-beta` and when they release the public one, we can add that also?

**Comment by ran-dall** [comment]
> Could we call this package cloudflare-warp-beta and when they release the public one, we can add that also?

This would be the wrong place for it then, as the main HBC tap is for only for stable software. You'd need to open it here: https://github.com/Homebrew/homebrew-cask-versions

**Comment by miccal** [comment]
> This would be the wrong place for it then, as the main HBC tap is for only for stable software. 

[It is acceptable to have this submitted to the main repo.](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version)

**Review by reitermarkus — APPROVED** [review]

**Comment by ran-dall** [comment]
> > This would be the wrong place for it then, as the main HBC tap is for only for stable software.
> 
> [It is acceptable to have this submitted to the main repo.](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md?rgh-link-date=2020-09-04T01%3A43%3A37Z#but-there-is-no-stable-version)

There is a stable version, it's just not out yet. So I don't think this would apply.

**Pull request merged**